### DISCUSSION TITLE
How to do page wrap in flextable

### DISCUSSION THREAD

**Discussion by sainiamaan** [davidgohel/flextable#538] [Q&A] [answered]
How to do page wrap in flextable when my table is too wide and all columns don't fit on one page in word document. Is it possible to split table on a column.

**Comment by davidgohel** [comment, marked as answer]
no, sorry, that's not a feature of flextable.

You can reduce font size with `fit_to_width()` but if there is not enough room to show all columns of the table on a page, maybe that means it will be difficult to read and the layout should be changed. I can't imagine reading a table with some information spread across different pages.
